backend/src/apiserver/webhook/pipelineversion_webhook.go

```diff
@@ -101,7 +102,7 @@ func (p *PipelineVersionsWebhook) ValidateCreate(
 		return nil, newBadRequestError(fmt.Sprintf("The pipeline spec is invalid JSON: %v", err))
 	}
 
-	tmpl, err := template.NewV2SpecTemplate(pipelineSpec)
+	tmpl, err := template.NewV2SpecTemplate(pipelineSpec, *p.cacheEnabled)
```

Comment:
This just checks if the syntax is valid and doesn't submit the template so this can always be set to `false`.

Reply:
Done.